facet-poke/tests/enum.rs

```diff
@@ -97,3 +97,181 @@ fn build_enum() {
         );
     }
 }
+
+#[test]
```

Comment:
Both these tests are a full on copy-paste.

I could macroify these if that would be preferable?

Reply:
Well, in the latest main this test is entirely deleted so I guess that resolves my dilemma